SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialSender.cpp

```diff
@@ -1177,22 +1183,28 @@ void USpatialSender::AddPendingUnreliableRPC(UObject* TargetObject, const FPendi
 	{
 		UE_LOG(LogSpatialSender, Warning, TEXT("AddPendingUnreliableRPC: No connection for object %s (RPC %s, actor %s, entity %lld)"),
 			*TargetObject->GetName(), *Function->GetName(), *TargetActor->GetName(), TargetObjectRef.Entity);
-		return;
+		return false;
 	}
 
 	APlayerController* Controller = Cast<APlayerController>(OwningConnection->OwningActor);
 	if (Controller == nullptr)
 	{
 		UE_LOG(LogSpatialSender, Warning, TEXT("AddPendingUnreliableRPC: Connection's owner is not a player controller for object %s (RPC %s, actor %s, entity %lld): connection owner %s"),
 			*TargetObject->GetName(), *Function->GetName(), *TargetActor->GetName(), TargetObjectRef.Entity, *OwningConnection->OwningActor->GetName());
-		return;
+		return false;
+	}
+
+	USpatialActorChannel* ControllerChannel = NetDriver->GetOrCreateSpatialActorChannel(Controller);
+	if (!(ControllerChannel && ControllerChannel->IsListening()))
```

Comment:
```suggestion
	if (ControllerChannel == nullptr || !ControllerChannel->IsListening())
```

Reply:
This is just my preference, either way is valid.

Reply:
I like it!

Reply:
Need to fix a bug now :D

Reply:
ah, it's all right!